test_project/admin.py

```diff
@@ -0,0 +1,35 @@
+import django.contrib.auth.admin
+import django.contrib.redirects.admin
+import django.contrib.sites.admin
+import djangoplicity.actions.admin
+from djangoplicity.contrib.admin.discover import autoregister
+from djangoplicity.contrib.admin.sites import AdminSite
+
+import djangoplicity.mailinglists.admin
+import djangoplicity.newsletters.admin
+
+# Register each applications admin interfaces with
+# an admin site.
+admin_site = AdminSite(name="admin_site")
+adminlogs_site = AdminSite(name="adminlogs_site")
+adminshop_site = AdminSite(name="adminshop_site")
+
+autoregister(admin_site, django.contrib.auth.admin)
+autoregister(admin_site, django.contrib.sites.admin)
+
+autoregister(admin_site, djangoplicity.mailinglists.admin)
```

Comment:
Are you sure this file is being used?. Because I don't see this being used in the urls.py file, besides this is not necessary here. You can just register the models to the default admin site:

https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#modeladmin-objects

Reply:
I'll check it out